pulp_2to3_migration/app/models/base.py

```diff
@@ -478,7 +478,7 @@ def set_distributors(cls, repo_id, repo_type, distributor_repo_ids):
                     pulp2_resource_type=cls.DISTRIBUTOR,
                     pulp2_repo_type=repo_type,
                     pulp2_repo_id=repo_id,
-                    pulp2_resource_repo_id=distributor_repo_id,
+                    pulp2_resource_repo_id=distributor_repo_id or '',
```

Comment:
I do not believe it's realistic to have no distributors but I added just in case.

Reply:
Is there anywhere else we set this that might have the same issue? or just the two uses in this file?

Reply:
N<, I did my own search, we're good :)